### ISSUE TITLE
Build Warnings

### ISSUE THREAD

**Issue by joe9** [9fans/plan9port#63]
```
>>> cd /home/j/dev/apps/plan9/plan9port/src/cmd/eqn; mk all
9c  input.c
   fgets(ebuf, sizeof ebuf, curfile->fin);
   ^
>>> cd /home/j/dev/apps/plan9/plan9port/src/cmd/fossil; mk all
9c  fossil.c
/usr/include/features.h:148:3: warning: #warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE" [-Wcpp]
 # warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE"
   ^
>>> cd /home/j/dev/apps/plan9/plan9port/src/cmd/fossil; mk all
9c  9p.c
/usr/include/features.h:148:3: warning: #warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE" [-Wcpp]
 # warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE"
   ^
>>> cd /home/j/dev/apps/plan9/plan9port/src/cmd/fossil; mk all
9c  9auth.c
/usr/include/features.h:148:3: warning: #warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE" [-Wcpp]
 # warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE"
   ^
>>> cd /home/j/dev/apps/plan9/plan9port/src/cmd/fossil; mk all
9c  9dir.c
/usr/include/features.h:148:3: warning: #warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE" [-Wcpp]
 # warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE"
   ^
>>> cd /home/j/dev/apps/plan9/plan9port/src/cmd/fossil; mk all
9c  9excl.c
/usr/include/features.h:148:3: warning: #warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE" [-Wcpp]
 # warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE"
   ^
>>> cd /home/j/dev/apps/plan9/plan9port/src/cmd/fossil; mk all
9c  9fid.c
/usr/include/features.h:148:3: warning: #warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE" [-Wcpp]
 # warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE"
   ^
>>> cd /home/j/dev/apps/plan9/plan9port/src/cmd/fossil; mk all
9c  9fsys.c
/usr/include/features.h:148:3: warning: #warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE" [-Wcpp]
 # warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE"
   ^
>>> cd /home/j/dev/apps/plan9/plan9port/src/cmd/fossil; mk all
9c  9lstn.c
```

**Comment by 0intro** [comment]
This issue has been reported and fixed, but the fix hasn't made its way into the repository yet.

You can apply this patch: https://plan9port-review.googlesource.com/#/c/1500/

**Comment by joe9** [comment]
@0intro The fix was made a few months ago. Is there a more updated repo than this? Or, does the process normally take this long?

Thanks

**Comment by 0intro** [comment]
The CL has been merged to the repository as commit [657f699](https://github.com/9fans/plan9port/commit/657f699ef7e5a91a529337cac76ed48c68a28583).
